content/articles/caa-record.markdown

```diff
@@ -14,13 +14,13 @@ categories:
 
 ---
 
-## What is a CAA record?
+## What's a CAA record?
```

Comment:
This article is part of a series about specific record types. I think if we make this change here, we should apply it to all the other types e.g. https://github.com/dnsimple/dnsimple-support/blob/master/content/articles/cname-record.markdown

Basically, these articles should likely all have the same structure. Do you agree @itsalyse ?

Reply:
We should. We're using "don't" and "won't" and "I'm" in a lot of others, so "what's" would also be used. I've been changing them as I go through the support site articles.